src/batou/tests/test_repository.py

```diff
@@ -14,11 +14,11 @@ def test_repository_hg_norepo(tmpdir):
     environment.base_dir = tmpdir
     os.chdir(tmpdir)
 
-    with pytest.raises(batou.utils.CmdExecutionError) as e:
+    with pytest.raises(batou.utils.CmdExecutionError):
         MercurialRepository(environment)
 
     subprocess.check_call(['hg', 'init'])
-    repository = MercurialRepository(environment)
+    MercurialRepository(environment)
```

Comment:
```suggestion
    # No error after initialisation.
    MercurialRepository(environment)
```
If we do not write an explicit assertNothingRaises a comment might help for better understanding.